### PR TITLE
[SCHEMA] Common error messages for rules that have no check in schema.

### PR DESCRIPTION
**Pull request by rwblair**
Modification of  #1114

### CODE REVIEW

File: src/schema/rules/errors.yaml [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+# BIDS Validator Original Issue Code #26
+NiftiHeaderUnreadable:
+    code: NIFTI_HEADER_UNREADABLE
+    message: |
+        We were unable to parse header data from this NIfTI file.
+        Please ensure it is not corrupted or mislabeled.
+    level: error
+    selectors:
+    - extension in [".nii", ".nii.gz"]
```
Comment (effigies) [inline src/schema/rules/errors.yaml:12]: Does this syntax need to be updated?

Comment (rwblair) [reply src/schema/rules/errors.yaml]: yes

**Review by effigies — APPROVED** [review]

File: src/schema/rules/errors.yaml [hunk 2]
```diff
@@ -0,0 +1,252 @@
+---
+# This file describes rules for broad "standard" errors in the specification.
+
+# BIDS Validator Original Issue Code #26
+NiftiHeaderUnreadable:
+    code: NIFTI_HEADER_UNREADABLE
```
Comment (yarikoptic) [inline src/schema/rules/errors.yaml:6]: Me curious: Why code isn't just the key of the entry?

Comment (rwblair) [reply src/schema/rules/errors.yaml:6]: The idea was towards consistency, there are places in the schema that have custom error messages use the `code` and `message` keys:
https://github.com/bids-standard/bids-specification/blob/a93d4f2a27b2b79ba569a1d02f1e7f560af3525f/src/schema/rules/sidecars/asl.yaml#L27

Though those are nested in that `issue` key. I don't know if we want to consider lifting the `code` and `message` there a level higher or not. The schema needs its own schema.

**Review by sappelhoff — APPROVED** [review]

**Comment by sappelhoff** [comment]
I think I should have rebased this before merging. The changes lead to a broken CI in `master` now -- can you have a look @rwblair?

**Pull request merged**